docs/config-metrics.md

```diff
@@ -0,0 +1,7 @@
+# Config Metrics
+
+| Metric name| Metric type | Labels/tags | Status |
+| ---------- | ----------- | ----------- | ----------- |
+| openshift_cluster_infrastructure_provider | Gauge | `config`=&lt;config-name&gt; <br> `namespace`=&lt;config-namespace&gt; | STABLE |
```

Comment:
the labels for this are [`type` and `region`](https://github.com/openshift/cluster-kube-apiserver-operator/blob/c7b1e077d913fbe6e89d4c10312346234e31e4e4/pkg/operator/configmetrics/configmetrics.go#L22)